kratos/geometries/brep_curve_on_surface.h

```diff
@@ -238,6 +238,21 @@ class BrepCurveOnSurface
         return mpCurveOnSurface;
     }
 
+    ///@}
+    ///@name Curve Properties
+    ///@{
+
+    /* @brief Provides intersections of the nurbs curve with the knots of the surface,
+     *         using the interval of this curve.
+     * @param vector of span intervals.
+     * @param index of chosen direction, for curves always 1.
+     */
+    void Spans(std::vector<double>& rSpans, IndexType DirectionIndex = 1) const
```

Comment:
I know that for curve on surface you don't use it but for correct documentation the direction should be 0. Isn't it?

Reply:
Yes, you're right!